APIAssignment/config.properties

```diff
@@ -1 +1 @@
-baseURL=https://automationexercise.com/api/productsList
\ No newline at end of file
+baseURI=https://automationexercise.com/api/
```

Comment:
create config package for .properties file

Reply:
Done !